virtManager/addhardware.py

```diff
@@ -860,7 +860,11 @@ def _build_redir_type_combo(self):
 
 
     def _build_panic_model_combo(self):
-        values = [[None, _("Hypervisor default")]]
+        guest = self.vm.get_xmlobj()
+        values = []
+        for m in guest.lookup_domcaps().supported_panic_models():
+            values.append([m, m])
+
```

Comment:
For the same reason we should keep the `Hypervisor default` and set it as default_value.

Reply:
I've added a line that will set `Hypervisor default` as the default value and pushed it. Once tests are done I'll merge this. Thanks!